src/scripts/utilities.js

```diff
@@ -134,6 +142,13 @@ If you remove your bookmark, the DM you received is removed
 
 Any message the bot sends via DM can be deleted by applying an 'x' (❌) reaction to it
 					`
+				}, {
+					name: 'Roles',
+					value: `
+Moderators can create a self-assigned role message using the \`!role message\` command
+
+Use \`!help role message\` for more information
```

Comment:
I think messages should be separated from code
https://github.com/chris-mcdonald-dev/korean-discord-app/tree/development/src/constants